examples/microprofile/openapi-basic/pom.xml

```diff
@@ -44,6 +44,10 @@
             <groupId>io.helidon.microprofile.openapi</groupId>
             <artifactId>helidon-microprofile-openapi</artifactId>
         </dependency>
+        <dependency>
```

Comment:
Is this example using both Jackson and Jsonb ? Is it possible to make it work on either but not both ?

Reply:
This example uses only JSON-B. I added` jackson` dependency because `helidon-microprofile-openapi` depends on `smallrye-open-api-core` where `jackson` is excluded so we use the latest version of `jackson`. 
I think `jackson` dependency was not there before because tests were disabled and the issue did not show up.

Reply:
As discussed, I removed the changes and I am going to open another issue to fix disabled tests.